monitor/remoteapi/server/server_test.go

```diff
@@ -115,15 +115,15 @@ func TestValidateTypes(t *testing.T) {
 			So(err, ShouldNotBeNil)
 		})
 
-		Convey("If the event name has bad charaters, it should error.", func() {
+		Convey("If the event name has utf8 charaters, it should succeed.", func() {
 			event := &common.EventInfo{
 				EventType: common.EventStart,
 				PUType:    common.ContainerPU,
-				Name:      "^^^",
+				Name:      "coolguy@gmail.com",
```

Comment:
Can you try all the utf-8 characters here ? Check our unit tests in gaia

Reply:
Done.